example/discovery.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:dbus/dbus.dart';
+import 'package:bluez/bluez.dart';
+
+import 'package:convert/convert.dart';
+
+void process_ble(var device) {
+  var mac = device.address;
+  int rssi = device.rssi;
+  print('\nName: ${device.name}');
+  print('Address: ${mac}\nRSSI: ${rssi}');
```

Comment:
This would be clearer splitting the print into multiple lines instead of using the `\n`.